frontend/src/pages/destructionlist/detail/pages/process-review/DestructionListProcessReviewPage.tsx

```diff
@@ -173,32 +173,31 @@ export function DestructionListProcessReviewPage() {
       )?.label;
 
     const tooltipData = {
-      archiefactiedatum,
-      oud_archiefactiedatum: zaak?.archiefactiedatum,
-      selectielijstklasse: getLabel(selectielijstklasse),
-      oud_selectielijstklasse: getLabel(zaak.selectielijstklasse),
-      opmerking: comment,
+      "Archiefactiedatum (nieuw)": archiefactiedatum,
+      "Archiefactiedatum (oud)": zaak?.archiefactiedatum,
+      "Selectielijstklasse (nieuw)": getLabel(selectielijstklasse),
+      "Selectielijstklasse (oud)": getLabel(zaak?.selectielijstklasse),
+      Opmerking: comment,
     };
 
     // Filtering logic:
-    // 1. Keep 'oud_' values only if the new value exists and is different.
+    // 1. Keep 'oud' values only if the new value exists and is different.
     // 2. Remove empty, null, or undefined values.
     // 3. Remove duplicate values.
     const filteredTooltipData = Object.fromEntries(
       Object.entries(tooltipData).filter(([key, value], _, arr) => {
-        // 1.
-        if (key.startsWith("oud_")) {
-          const newKey = key.replace("oud_", "");
+        // 1
+        if (key.includes("(oud)")) {
+          const newKey = key.replace("(oud)", "(nieuw)").trim();
```

Comment:
I think acting on the presentation (translated strings) is a bit of an antipattern compared to simply checking the source data. Is there a reason why we can't simply do these check when construction tooltipdata (acting on the data)?

Let's do as little string parsing/replacing as possible.